charts/telegraf-operator/values.yaml

```diff
@@ -1,6 +1,7 @@
 replicaCount: 3
 image:
   repository: quay.io/influxdb/telegraf-operator
+  tag: 1.2.0
```

Comment:
When you change to this:
```
    # Overrides the image tag whose default is the Helm Chart appVersion.
    tag: ""
```
maintainers don't need to remember about changing tag when AppVersion is changed.